bin/ch-run.c

```diff
@@ -6,70 +6,26 @@
 #define _GNU_SOURCE
 #include <argp.h>
 #include <assert.h>
-#include <errno.h>
-#include <fcntl.h>
-#include <libgen.h>
-#include <sched.h>
-#include <semaphore.h>
 #include <stdbool.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/mman.h>
-#include <sys/mount.h>
-#include <sys/prctl.h>
-#include <sys/stat.h>
-#include <sys/syscall.h>
-#include <time.h>
 #include <unistd.h>
 
 #include "charliecloud.h"
 
-/** Constants and macros **/
-
-/* Host filesystems to bind. */
-#define USER_BINDS_MAX 10
-const char * DEFAULT_BINDS[] = { "/dev",
-                                 "/etc/passwd",
-                                 "/etc/group",
-                                 "/etc/hosts",
-                                 "/etc/resolv.conf",
-                                 "/proc",
-                                 "/sys",
-                                 NULL };
-
-/* Number of supplemental GIDs we can deal with. */
-#define SUPP_GIDS_MAX 128
 
-/* Maximum length of paths we're willing to deal with. (Note that
-   system-defined PATH_MAX isn't reliable.) */
-#define PATH_CHARS 4096
+/** Constants and macros **/
 
-/* Log the current UIDs. */
-#define LOG_IDS log_ids(__func__, __LINE__)
+/* Maximum number of user binds. */
```

Comment:
Can we remove this comment? USER_BINDS_MAX[] is not present in charliecloud.c and was removed from ch-run.c

Reply:
Done.